projects/igniteui-angular/src/lib/grids/grid-navigation.service.ts

```diff
@@ -138,6 +138,9 @@ export class IgxGridNavigationService {
         }
         if (!this.activeNode || !Object.keys(this.activeNode).length || this.activeNode.row < 0 || this.activeNode.row > gridRows - 1) {
             const hasLastActiveNode = Object.keys(this.lastActiveNode).length;
+            if (hasLastActiveNode && this.grid.cellSelection !== GridSelectionMode.none && !this.grid.selectionService.isInMap(this.lastActiveNode)) {
+                return;
+            }
```

Comment:
Why this change is necessary? With this change currently the default tab navigation through the grid is broken:
![TABGrid](https://user-images.githubusercontent.com/34240583/220622714-36ce15d7-792f-495c-8d19-bfa3e02f38e1.gif)
![TABMCHGrid](https://user-images.githubusercontent.com/34240583/220622728-b26dcdc5-ce75-40b5-8c4a-7dd28ce74e7a.gif)